pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -1,14 +1,12 @@
-{ stdenv, fetchFromGitHub, curl, dmd, gcc }:
+{ stdenv, fetchurl, curl, dmd, gcc }:
 
 let version = "0.9.24"; in
 stdenv.mkDerivation {
   name = "dub-${version}";
 
-  src = fetchFromGitHub {
-    sha256 = "52ea88d075f3a43c84d8cdc3a8683bcc8133b7db";
-    rev = "v${version}";
-    repo = "dub";
-    owner = "dlang";
+  src = fetchurl {
```

Comment:
This is the same thing only worse; the point of `fetchFromGitHub` is to make it convenient to use github's generated tarballs while also being robust wrt. changes to the way the tarballs are generated (e.g., compression levels).